src/pages/registration/RegistrationFormActions.tsx

```diff
@@ -52,22 +63,23 @@ export const RegistrationFormActions = ({ tabNumber, setTabNumber, validateForm
       const newErrors = validateForm(updateRegistrationResponse.data);
       setTouched(setNestedObjectValues(newErrors, true));
       dispatch(setNotification({ message: t('feedback.success.update_registration'), variant: 'success' }));
+      if (isLastTab) {
+        history.push(getRegistrationLandingPagePath(values.identifier));
+      }
```

Comment:
Vi trenger sikkert ikke å håndtere form errors når vi uansett skal navigere bort fra siden? (tror valideringen som vises på landing page er uanvhengig av dette, men det burde dobbeltsjekkes)
```suggestion
      if (isLastTab) {
        history.push(getRegistrationLandingPagePath(values.identifier));
      } else {
        const newErrors = validateForm(updateRegistrationResponse.data);
        setTouched(setNestedObjectValues(newErrors, true));
        dispatch(setNotification({ message: t('feedback.success.update_registration'), variant: 'success' }));
      }
```